erts/emulator/beam/erl_bif_info.c

```diff
@@ -92,8 +92,12 @@ static char erts_system_version[] = ("Erlang/OTP " ERLANG_OTP_RELEASE
 				     " [source]"
 #endif
 #endif	
-#ifdef ARCH_64
+#if defined(ARCH_64)
 				     " [64-bit]"
+#elif defined(ARCH_32)
+                                     " [32-bit]"
```

Comment:
Nitpick: this file is using a whitespace format based on tabs, it would be great to keep it consistent on L98